launch/launch/logging/__init__.py

```diff
@@ -407,25 +441,16 @@ def __init__(self, *args, **kwargs):
         self.propagate = False
 
 
-default_log_dir = _make_unique_log_dir(
-    base_path=os.path.join(os.path.expanduser('~'), '.ros/log')
-)
-
-
 def reset():
     """Reset logging."""
     # Reset existing logging infrastructure
     for logger in LaunchLogger.all_loggers:
         logger.setLevel(logging.NOTSET)
         del logger.handlers[:]
-    # Back to default logging setup
-    launch_config.log_dir = None
-    launch_config.file_handlers = {}
-    launch_config.screen_handler = None
-    launch_config(
-        level=logging.INFO, log_dir=default_log_dir,
-        log_format='default', screen_format='default'
-    )
+    launch_config.reset()
+    launch_config.level = logging.INFO
+    launch_config.set_screen_format('default', None)
+    launch_config.set_log_format('default', None)
     logging.setLoggerClass(LaunchLogger)
```

Comment:
@clalancette hmm I wonder if `launch_config.reset()` shouldn't already set reasonable defaults.

Reply:
Yeah, that's a good point.  I'll switch to that, I like it better.